app/views/editor/modal/SaveVersionModal.coffee

```diff
@@ -24,6 +24,8 @@ module.exports = class SaveVersionModal extends ModalView
     @model = options.model or options.level
     @isPatch = not @model.hasWriteAccess()
     @hasChanges = @model.hasLocalChanges()
+    if utils.isOzaria
+      @commitMessage = options.commitMessage
```

Comment:
This seems like the kind of thing that should have merged behavior rather than feature-flagged behavior; at the very least, if the `commitMessage` is not included, we can just ignore it, but don't need `isOzaria` checks here, right?